inspirehep/base/templates/search/easy_search.html

```diff
@@ -0,0 +1,222 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+ <div id="easy-search" class="collapse">
+  <form class="form-horizontal">
+  <div class="form-group">
+    <label for="inputAuthor" class="col-sm-2 control-label">Author</label>
```

Comment:
For consistency and avoiding overriding potentially similar selectors, shall we add a prefix to all the classes and ids used in the easy search? Say `easy-search-author`, `easy-search-title` etc.